copyright

```diff
@@ -1605,6 +1600,15 @@ Files:
 Copyright: Gefüllte Taubenbrust <jeaminer23@gmail.com>
 License: CC-BY-SA-4.0
 
+Files:
+ images/effect/archonteleport/*
+ images/effect/archonbteleport/*
+ images/effect/archoncteleport/*
+ images/effect/archondteleport/*
+Copyright: Gefüllte Taubenbrust <jeaminer23@gmail.com>
+License: CC-BY-SA-4.0
+Comment: Derived from works by @Karirawri (under the same license).
```

Comment:
If they're derived from Karirawri, and Karirawri's are derived from MZ and I, should this be derived from us two as well?

Reply:
I have no idea tbh